cmd/celestia-appd/cmd/gas_price_updater.go

```diff
@@ -0,0 +1,70 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/celestiaorg/celestia-app/v5/app"
+	"github.com/celestiaorg/celestia-app/v5/app/params"
+	"github.com/celestiaorg/celestia-app/v5/pkg/appconsts"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+const (
+	// Old default minimum gas price that needs to be updated
+	oldDefaultMinGasPrice = "0.002utia"
+)
+
+// checkAndUpdateMinGasPrices checks if the minimum gas prices in app.toml
+// are set to the old default value and updates them if necessary.
+func checkAndUpdateMinGasPrices(cmd *cobra.Command) error {
+	// Get the home directory from the command
+	homeDir, err := cmd.Flags().GetString(flags.FlagHome)
+	if err != nil {
+		return fmt.Errorf("failed to get home directory: %w", err)
+	}
+	if homeDir == "" {
+		homeDir = app.NodeHome
+	}
```

Comment:
[optional] this package imports app which has NodeHome so I think it should be possible to remove these lines and instead use `NodeHome` directly

Ref: https://github.com/celestiaorg/celestia-app/blob/dcded42f669298c549b10d31bf31a364cbe240a4/app/init.go#L22-L24